.github/workflows/build-push.yaml

```
@@ -18,7 +18,7 @@ jobs:
       - name: Install Go 1.19
```

```suggestion
      - name: Set up Golang
```

nit:
Let's change this to a generic name.
Other workflows also need the same change.